### DISCUSSION TITLE
Analysing simulation outputs

### DISCUSSION THREAD

**Discussion by MartijnOW** [asreview/asreview#1241] [Q&A] [answered]
Dear ASReview community,

When performing a simulation in the new Makita extension, I get the results of each simulation as a .asreview file in my output folder. Now I come to the analysis part, I am wondering whether there is a efficient way to load the outputs of the simulations to perform some analysis on variation. Preferably, what I would like (in the end) is a table or dataframe with two colums per simulation, the first column with the row number of the labelled dataset and the second column with the order within ASReview. By opening the .asreview file with the metrics function in the command prompt, a lot of data is given with many brackets in between, making it hard to efficiently collect only the data I would like. Is there a more efficient way to show the outcomes of the simulation?

Kind regards,
Martijn

**Comment by Rensvandeschoot** [comment, marked as answer]
You could try the `merge_descriptives` and `merge_metrics` scripts available in the Makita repository: https://github.com/asreview/asreview-makita#add-and-use-scripts.
